cds_ils/authentication/views.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020 CERN.
+#
+# CDS-ILS is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+
+"""CDS-ILS authentication views module."""
+
+from flask import Blueprint, current_app, redirect
+
+cern_oauth_blueprint = Blueprint("cern_openid_oauth", __name__)
+
+
+@cern_oauth_blueprint.route('/cern_openid/logout/')
+def logout_redirect():
+    """Redirect to the CERN OpenID logout."""
+    return redirect(current_app.config['OAUTH_REMOTE_REST_APP']['logout_url'])
```

Comment:
Is this still needed? Wasn't it enough to define the `SECURITY_ ...` var with the url?

Reply:
it is needed necause flask security expects the urls within the application host, so redirect to keycloak is marked as invalid :)